src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -202,7 +203,18 @@ class SaveInProgressIntro extends React.Component {
         retentionPeriodStart,
         unauthStartText,
       } = this.props;
-      const unauthStartButton = (
+      const CustomLink = this.props.customLink;
```

Comment:
I've seen typical convention to PascalCase component even for props, e.g this.props.CustomLink